The value 1.0 without f is a double constant and gcc6 complains
about a not existing template for max(double,float).

--- libs/ardour/parameter_descriptor.cc	2016-10-04 09:19:13.780997674 +0200
+++ libs/ardour/parameter_descriptor.cc	2016-10-04 09:19:45.237999989 +0200
@@ -178,8 +178,8 @@
 			largestep = largestep / logf(30.0f);
 		} else if (integer_step) {
 			smallstep = 1.0;
-			step      = std::max(1.0, rint(step));
-			largestep = std::max(1.0, rint(largestep));
+			step      = std::max(1.0f, rint(step));
+			largestep = std::max(1.0f, rint(largestep));
 		}
 	}
 }
